src/Entity/User.php

```diff
@@ -33,6 +33,36 @@ class User extends BaseUser
      */
     private $portalApps = [];
 
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $UserFullName;
```

Comment:
Properties should be written in camelCase.
There is no need to prefix properties in the `User` class with `User` – we already now that they are `User` properties.
This property should be called `$fullName` and the same goes for the rest of the new properties.

Reply:
fixed camelCake